sycl/test-e2e/Matrix/joint_matrix_rowmajorA_rowmajorB.cpp

```diff
@@ -15,6 +15,8 @@
 // RUN: %if gpu %{ env IGC_JointMatrixLoadStoreOpt=1 %{run} %t.out %}
 // RUN: %if gpu %{ env IGC_JointMatrixLoadStoreOpt=0 %{run} %t.out %}
 
+// XFAIL: gpu-intel-dg2
+
```

Comment:
Same as above:
Adding XFAIL will make the code fail silently for the other supported combination.